### ISSUE TITLE
Resolve "aria-* attribute is defined in WAI-ARIA" feedback

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag-act#452]
### Sailesh Panchang

- [ ] Why is this rule under consideration as it states: "Accessibility Requirements Mapping: This rule is not required for conformance." An incorrect ARIA-* attribute nothing for accessibility testing. If an element has aria-unchecked="true", that element will get flagged while doing a test for SC 4.1.2 (manual / automated) as it may not have its state exposed (assuming state is not exposed via a correct property). And if it has another correct property set, presence of aria-unchecked is just a poor code, a matter of code quality. Such coding issues do not have to be highlighted by a tool. (Note: during a recent call this view was highlighted too: best practices for coding is not what this effort is about).

> **Answer**: Most folks in the task force do seem to think best practice rules are useful, although they have a lower priority than WCAG conformance rules

- [ ] Perhaps a different example should be used if this rule is retained in place of passed-example#3. An alert does not need aria-live=assertive. `<div role="alert" aria-live="assertive"> Your session will expire in 60 seconds.</div>`

### Charu Pandhi

- [ ] Should there be more passing examples with HTML elements with implicit roles for example `<button>` has role of a button and can use all the attributes allowed for button role.

- [ ] I would think it would map to 4.1.2 WCAG rec, if not what?

**Comment by jkodu** [comment]
Replies to unanswered Q's:

## Sailesh Panchang

- [x] Perhaps a different example should be used if this rule is retained in place of passed-example#3. An alert does not need aria-live=assertive. <div role="alert" aria-live="assertive"> Your session will expire in 60 seconds.</div>
> Updated the example - See - https://github.com/act-rules/act-rules.github.io/pull/1323/files#diff-a739e219780be6416bfad963fd99f519R61

## Charu Pandhi
- [x] Should there be more passing examples with HTML elements with implicit roles for example <button> has role of a button and can use all the attributes allowed for button role.
> I added one example (passed example 5), but I think adding more of the same kind of examples is not necessary.

- [x] I would think it would map to 4.1.2 WCAG rec, if not what?
> Several detailed discussions have been had on this @cpandhi, for example see https://github.com/act-rules/act-rules.github.io/issues/353. ACT-R made a conscious decision to not map this rule to any SC.

**Comment by maryjom** [comment]
[Survey results](https://www.w3.org/2002/09/wbs/93339/ACTARIAAttribute2/results) completed on 22 July and discussed in 23 July meeting.
CFC to call this rule complete went out to TF on 23 July to be completed 27 July.

**Comment by maryjom** [comment]
CFC approved sending this to the AG WG - completed 27 July with only +1 votes.
